remindme-common/src/main/java/it/andreafailli/remindme/Profiles.java

```diff
@@ -1,9 +1,7 @@
 package it.andreafailli.remindme;
 
-public class Profiles {
+public interface Profiles {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)